include/seqan3/range/views/async_input_buffer.hpp

```diff
@@ -435,7 +435,7 @@ namespace seqan3::view
  * |                                           |                                   |                                   |
  * | std::iterator_traits \::iterator_category |                                   | *none*                            |
  *
- * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
+ * See the \link views views submodule documentation \endlink for detailed descriptions of the view properties.
```

Comment:
Hm, I think it's ok like this. Was like this before, too...

Reply:
> Hm, I think it's ok like this. Was like this before, too...

I was just as picky as possible about everything that touches these things, since there won't be many other PR's where you see these kind of things so prominently. So better yet to fix them then later. But, that can be done later or never. Not so important